ng build --prod throwing exception

<!--🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅

Oh hi there! 😄

To expedite issue processing please search open and closed issues before submitting a new one.
Existing issues often contain information about workarounds, resolution, or progress updates.

🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅-->


# 🐞 Bug report

### Command (mark with an `x`)
<!-- Can you pin-point the command or commands that are effected by this bug? -->
<!-- ✍️edit: -->
```
- [ ] new
- [x ] build
- [ ] serve
- [ ] test
- [ ] e2e
- [ ] generate
- [ ] add
- [ ] update
- [ ] lint
- [ ] xi18n
- [ ] run
- [ ] config
- [ ] help
- [ ] version
- [ ] doc
```

### Is this a regression?
No


### Description
Building production for angular crashes because of Terser

## 🔬 Minimal Reproduction
Create a new angular app with "ng new my-app" as described on the angular.io get started
page and it will crash.

## 🔥 Exception or Error
<pre><code>
`ERROR in common.162374785956ac8f24a6.js from Terser
TypeError: Cannot read property 'minify' of undefined
    at minify (C:\path\node_modules\terser-webpack-plugin\dist\minify.js:162:59)
    at module.exports (C:\path\node_modules\terser-webpack-plugin\dist\worker.js:16:41)
    at handle (C:\path\node_modules\worker-farm\lib\child\index.js:44:8)
    at process.<anonymous> (C:\path\node_modules\worker-farm\lib\child\index.js:51:3)
    at process.emit (events.js:182:13)
    at emit (internal/child_process.js:812:12)
    at process._tickCallback (internal/process/next_tick.js:63:19)
`

</code></pre>


## 🌍 Your Environment
<pre><code>
$ ng --version

     _                      _                 ____ _     ___
    / \   _ __   __ _ _   _| | __ _ _ __     / ___| |   |_ _|
   / △ \ | '_ \ / _` | | | | |/ _` | '__|   | |   | |    | |
  / ___ \| | | | (_| | |_| | | (_| | |      | |___| |___ | |
 /_/   \_\_| |_|\__, |\__,_|_|\__,_|_|       \____|_____|___|
                |___/
    

Angular CLI: 7.1.4
Node: 10.13.0
OS: win32 x64
Angular: 7.1.4
... animations, cli, common, compiler, compiler-cli, core, forms
... language-service, platform-browser, platform-browser-dynamic
... router

Package                           Version
-----------------------------------------------------------
@angular-devkit/architect         0.11.4
@angular-devkit/build-angular     0.11.4
@angular-devkit/build-optimizer   0.11.4
@angular-devkit/build-webpack     0.11.4
@angular-devkit/core              7.1.4
@angular-devkit/schematics        7.1.4
@ngtools/webpack                  7.1.4
@schematics/angular               7.1.4
@schematics/update                0.11.4
rxjs                              6.3.3
typescript                        3.1.6
webpack                           4.23.1
</code></pre>

**Anything else relevant?**

I also face this error when I run `ng build --prod --build-optimizer`

```
[ERROR] ERROR in 0.41a9786d57a2f2254213.js from Terser
[INFO] Time: 61783ms
[ERROR] TypeError: Cannot read property 'minify' of undefined
[INFO] chunk {0} 0.41a9786d57a2f2254213.js () 47.1 kB  [rendered]
```

I also face this error when I run `ng build --prod --build-optimizer`

```
[ERROR] ERROR in 0.41a9786d57a2f2254213.js from Terser
[INFO] Time: 61783ms
[ERROR] TypeError: Cannot read property 'minify' of undefined
[INFO] chunk {0} 0.41a9786d57a2f2254213.js () 47.1 kB  [rendered]
```

--- a/packages/angular_devkit/build_angular/package.json
+++ b/packages/angular_devkit/build_angular/package.json
@@ -44,7 +44,7 @@
     "stylus": "0.54.5",
     "stylus-loader": "3.0.2",
     "tree-kill": "1.2.1",
-    "terser-webpack-plugin": "1.2.1",
+    "terser-webpack-plugin": "1.2.2",
     "webpack": "4.29.0",
     "webpack-dev-middleware": "3.5.1",
     "webpack-dev-server": "3.1.14",

